### ISSUE TITLE
CVisuallyHidden is missing documentation

### ISSUE THREAD

**Issue by philwolstenholme** [chakra-ui/chakra-ui-vue#411]
We have the [CVisuallyHidden](https://github.com/chakra-ui/chakra-ui-vue/blob/c401a75559a8ea926fc568ff5b82dc0fa4485d2b/packages/chakra-ui-core/src/CVisuallyHidden/CVisuallyHidden.js) component but I couldn't find it mentioned on the documentation.

It'd be good to expose a documentation page for the component so people are aware Chakra provides one and don't end up making their own.

**Comment by codebender828** [comment]
Good point. At the time I didn't think to write about it because was only being used inside the library. But it seems people are using it so we should :D

At the moment I'm a little swamped with cranking out v1. Would you like to send in a PR for this page?

**Comment by philwolstenholme** [comment]
@codebender828 I had a feeling it might have been an internal only tool :) I'll try and take a look at how the docs work and creating a page.

This is an off-topic question, but we've just started a new project at work using Chakra. Do you know what the upgrade process to v1 will be like if we wanted to make the switch?
